webserver/shared/models/common.ts

```diff
@@ -89,3 +91,20 @@ export type PageInfo = {
     hasNextPage: boolean;
   };
 };
+
+export enum Direction {
+  Buy = 'buy',
+  Sell = 'sell',
+};
+
+export enum Dex {
+  WingRiders = 'WingRiders',
+  SundaeSwap = 'SundaeSwap',
+  MinSwap = 'MinSwap',
+  Unknown = 'Unknown',
```

Comment:
Do we actually care about an "unknown" case? Not clear when this would ever be useful.

Maybe this was created purely for `valueToDex`? Wouldn't it be better to abort the query quickly on an unknown DEX instead of going through with a value we know will return empty?

The only thing I could think of is maybe you're trying to write the webserver query so that it works properly even if the database indexer knows about DEXs that the webserver doesn't know about, but I don't think the code supports this anyway

Reply:
Case common is totally removed